src/main.rs

```diff
@@ -18,8 +22,64 @@ fn main() -> Result<(), std::io::Error> {
                 Ok(_) => {}
                 Err(e) => eprintln!("{}", e),
             }
+        } else if tokenized.cmd() == "pwd" {
+            match builtins::pwd() {
+                Ok(pwd) => println!("{}", pwd),
+                Err(e) => eprintln!("{}", e),
+            }
+        } else if tokenized.cmd() == "exit" {
+            match builtins::exit() {
+                Ok(_) => continue,
+                Err(e) => eprintln!("{}", e),
+            }
+        } else if tokenized.cmd() == "echo" {
+            match builtins::echo(tokenized.args()) {
+                Ok(_) => {}
+                Err(e) => eprintln!("{}", e),
+            }
+        } else if tokenized.cmd() == "alias" {
+            match builtins::alias(tokenized.args(), &mut aliases) {
+                Ok(_) => {}
+                Err(e) => eprintln!("{}", e),
+            }
+        } else if let Some(alias_cmd) = aliases.get(tokenized.cmd()) {
```

Comment:
Wouldn't this be better off as a pre-processing step before we get to the switch statement?

Reply:
Yeah good point, save some time doing any of the switch statement